systemd-units/plymouth-poweroff.sh.in

```diff
@@ -1,6 +1,5 @@
 #!/bin/bash
 
 if [ "$1" = "poweroff" ]; then
-	@PLYMOUTH_CLIENT_DIR@/plymouth quit --retain-splash
-	sleep 1
+	/usb/bin/chvt 6; @PLYMOUTH_CLIENT_DIR@/plymouth quit --retain-splash
```

Comment:
of course it's not intentional. I'll push a new PR since I noticed something odd.